# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../clearsilver/python-setup.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./python/setup.py.old	2004-12-22 20:57:19.562049128 +0100
+++ ./python/setup.py	2004-12-22 20:58:05.504064880 +0100
@@ -69,6 +69,8 @@
 LIB_DIRS = expand_vars(LIB_DIRS, make_vars)
 LIBRARIES = expand_vars(LIBRARIES, make_vars)
 
+os.environ["CFLAGS"] = os.environ["CFLAGS"].replace(' -c', '')
+
 setup(name="clearsilver",
       version=VERSION,
       description="Python ClearSilver Wrapper",
